app/v2/views.py

```diff
@@ -0,0 +1,52 @@
+from flask import request, jsonify, make_response, abort
+import json 
+from flask_jwt_extended import (
+    jwt_required,
+    get_jwt_identity, create_access_token
+
+)
+from validate_email import validate_email
+from .models import User
+from . import v2
+
+
+@v2.route('/api/v2/auth/signup', methods = ['POST'])
+def create_user():
+    data = request.get_json()
+    username = data.get('username')
+    name = data.get('name')
+    email = data.get('email')
+    password = data.get('password')
+
+    if not all([username,name,email,password]):
+        return jsonify({"message" : "Please fill in all the fields"}), 400
+    if not validate_email(email):
+        return jsonify({"message" : "Please input a valid email"}), 400
+    return jsonify({"message" : "Account created successfully"}), 201
+
+    
+@v2.route('/api/v2/auth/login', methods = ['POST'])
+def login():
+    if not request.is_json:
+        return jsonify({"msg": "Missing JSON in request"}), 400
+    username = request.json.get('username', None)
```

Comment:
`get` method returns `None` by default so they is no need for specifying this.

Reply:
okay..will work on that. thank you